docs/integrations/dell.md

```diff
@@ -0,0 +1,150 @@
+| ecs://bucket-a/namespace-a | Use a prefix to access the data only in this specific namespace |
+
+When you provide the `warehouse`, the last / will be ignored. The `ecs://bucket-a` is same with `ecs://bucket-a/`.
+
+### Dependencies of runtime
+
+The Iceberg `runtime` jar supports different versions of Spark and Flink. If the version was not matched in the example, please check the related document of Spark and Flink.
+
+Even though the [Dell ECS client](https://github.com/EMCECS/ecs-object-client-java) jar is backward compatible, Dell ECS still recommends using the latest version of the client.
+
+### Spark
+
+For example, to use the Dell ECS catalog with Spark 3.2.1, you should create a Spark session like:
+
+```sh
+spark-sql --packages org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:0.14.0,com.emc.ecs:object-client-bundle:3.3.2 \
+    --conf spark.sql.catalog.my_catalog=org.apache.iceberg.spark.SparkCatalog \
+    --conf spark.sql.catalog.my_catalog.warehouse=ecs://bucket-a/namespace-a \
+    --conf spark.sql.catalog.my_catalog.catalog-impl=org.apache.iceberg.dell.ecs.EcsCatalog \
+    --conf spark.sql.catalog.my_catalog.ecs.s3.endpoint=http://10.x.x.x:9020 \
+    --conf spark.sql.catalog.my_catalog.ecs.s3.access-key-id=test \
+    --conf spark.sql.catalog.my_catalog.ecs.s3.secret-access-key=xxxxxxxxxxxxxxxx
+```
+
+```python
+from pyspark.sql import SparkSession
+
+spark = SparkSession.builder\
+    .config("spark.jars.packages", "org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:0.14.0,com.emc.ecs:object-client-bundle:3.3.2")\
+    .config("spark.sql.extensions", "org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions")\
+    .config("spark.sql.catalog.my_catalog", "org.apache.iceberg.spark.SparkCatalog")\
+    .config("spark.sql.catalog.my_catalog.warehouse", "ecs://bucket-a/namespace-a")\
+    .config("spark.sql.catalog.my_catalog.catalog-impl", "org.apache.iceberg.dell.ecs.EcsCatalog")\
+    .config("spark.sql.catalog.my_catalog.ecs.s3.endpoint", "http://10.x.x.x:9020")\
+    .config("spark.sql.catalog.my_catalog.ecs.s3.access-key-id", "test")\
+    .config("spark.sql.catalog.my_catalog.ecs.s3.secret-access-key", "xxxxxxxxxxxxxxxx")\
+    .getOrCreate()
+```
+
+Then, use `my_catalog` to access the data in ECS. You can use `SHOW NAMESPACES IN my_catalog` and `SHOW TABLES IN my_catalog` to fetch the namespaces and tables of the catalog.
+
+The related problems of catalog usage:
+
+1. The [pyspark.sql.SparkSession.catalog](https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.sql.SparkSession.catalog.html#pyspark.sql.SparkSession.catalog) won't access the 3rd-party catalog of Spark, so please use DDL SQL to list all tables and namespaces.
+
+
+### Flink
+
+For example, to use the Dell ECS catalog with Flink 1.14, you should create a Flink environment like:
+
+```sh
+wget https://repo1.maven.org/maven2/org/apache/iceberg/iceberg-flink-runtime-1.14/0.14.0/iceberg-flink-runtime-1.14-0.14.0.jar
```

Comment:
Since we still don't release the iceberg 0.14.0, so we can not verify & experience this Dell ECS integration work.  It's possible to `wget` the latest snapshot artifact from the maven repo ?

Reply:
Because the dependencies are also added in this commit. I don't know which version should I use?